code/api/src/seeders/3-crates.js

```diff
@@ -4,41 +4,23 @@ module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.bulkInsert('crates', [
       {
-        name: 'Clothes for Men',
-        description: 'A monthly supply of trendy clothes for men.',
+        name: 'Clothes',
+        description: 'A monthly supply of trendy clothes.',
         createdAt: new Date(),
         updatedAt: new Date()
       },
       {
-        name: 'Clothes for Women',
-        description: 'A monthly supply of trendy clothes for women.',
+        name: 'Accessories',
+        description: 'A monthly supply of trendy accessories.',
         createdAt: new Date(),
         updatedAt: new Date()
       },
       {
-        name: 'Accessories for Men',
-        description: 'A monthly supply of trendy accessories for men',
+        name: 'Clothes and Accessories',
```

Comment:
I like that you removed the extra seeders since including gender in the name seems irrelevant right now.